src/node_options.cc

```diff
@@ -433,6 +433,9 @@ void GetOptions(const FunctionCallbackInfo<Value>& args) {
       case kInteger:
         value = Number::New(isolate, *parser.Lookup<int64_t>(field, opts));
         break;
+      case kUInteger:
```

Comment:
There’s also a list of exported constants which includes the other enum values below, fwiw